libs/core/src/lib/list/list.component.scss

```diff
@@ -9,3 +9,12 @@
 .fd-list__secondary {
     pointer-events: all;
 }
+
+// REMOVE AFTER CLOSING https://github.com/SAP/fundamental-styles/issues/1588
```

Comment:
Can you open an issue in NGX, link it in styles so we can track it?

Reply:
https://github.com/SAP/fundamental-ngx/issues/3233